### DISCUSSION TITLE
Query

### DISCUSSION THREAD

**Discussion by Nakshatra05** [esin/esin#27] [Guestbook]
Hii.....how to follow many people at once.

**Comment by esin** [comment]
> Hii.....how to follow many people at once.

Hey
Just used this project: https://github.com/esin/ghfollow

**Reply by Nakshatra05** [reply]
Is it safe to use?

**Reply by esin** [reply]
Sure, but you should have enough projects and don't be greedy :)

**Reply by Nakshatra05** [reply]
Is there any project for unfollowing in bulk too?

**Reply by esin** [reply]
> Is there any project for unfollowing in bulk too?

Nope, didn't find something like that

**Reply by Nakshatra05** [reply]
Can someone get banned for using this project, just in case?

**Reply by esin** [reply]
> Can someone get banned for using this project, just in case?

Sure

**Reply by Nakshatra05** [reply]
Is there any project for mass unfollowing too?

**Reply by esin** [reply]
> Is there any project for mass unfollowing too?

Didn't meet :(
But I think it's not a very difficult task - just need to replace one method in ghfollow - from PUT to DELETE

**Reply by Nakshatra05** [reply]
Can you make a repo for that too?

**Reply by esin** [reply]
> Can you make a repo for that too?

Well, you can try - https://github.com/esin/ghunfollow
